sdk/nodejs/appservice/zMixins.ts

```diff
@@ -209,7 +209,7 @@ interface FunctionAppArgsBase {
     /**
      * A `site_config` object as defined below.
      */
-    readonly siteConfig?: pulumi.Input<inputs.appservice.FunctionAppSiteConfig>;
+    readonly siteConfig?: pulumi.Input<inputs.appservice.LinuxFunctionAppSiteConfig>;
```

Comment:
That's a bunch of breaking changes, right? Also, isn't Windows supported?

Reply:
My impression was that it's up to us what kind of Function is created. The `zMixin`'s are our code, and they create the function [internally](https://github.com/pulumi/pulumi-azure/blob/8a052441ca50d039653965ba3d2e7c0121916f46/sdk/nodejs/eventgrid/zMixins.ts#L114-L115).

Regarding breaking changes, yes, it will probably recreate the `EventGridCallbackSubscription`'s. The last commit migrating to LinuxFunctionApp is just an experiment to see whether it fixes the CI build failure. We'll either not include this change, or look at the breaking changes in more detail.

Reply:
This PR was merged without the LinuxFunctionApp changes.